catalogue/webapp/components/IIIFImage/IIIFImage.tsx

```diff
@@ -66,26 +66,25 @@ const IIIFImage: FunctionComponent<Props> = ({
   // The Nextjs Image component has an experimental 'raw' layout feature
   // which will be rendered as a single image element with no wrappers, sizers or other responsive behavior.
   // We may be able to use this in future but, until then, render our own img element.
-  if (layout === 'raw' || layout === 'true-raw') {
+  if (layout === 'raw') {
     return (
-      <img
-        src={
-          layout === 'true-raw'
-            ? image.contentUrl
-            : iiifImageTemplate(image.contentUrl)({
-                size: `${width},`,
-              })
-        }
-        srcSet={''}
-        sizes={sizesString}
-        alt={image.alt || ''}
-      />
+      <StyledImageContainer background={background}>
+        <img
+          src={convertIiifImageUri(image.contentUrl, width)}
+          srcSet={''}
+          sizes={sizesString}
+          alt={image.alt || ''}
+        />
+      </StyledImageContainer>
     );
   }
 
   if (layout === 'fixed') {
     return (
-      <StyledImageContainer background={background}>
+      <StyledImageContainer
+        background={background}
+        style={{ height: image.height }} // to not have styledComponents generate too many classes
```

Comment:
> // to not have styledComponents generate too many classes

What do you mean?

Reply:
Styled Component generates a unique classname for every style. Since this is an image grid, and each image has it's own aspect ratio, the height value will vary greatly, so to prevent SC from generating loads of similar styles, or one line styles just for the height, I pass them in the style object.

Reply:
Oh was that the console warning 

```
styled-components.browser.esm.js?8ffb:1 Over 200 classes were generated for component ImageCard__StyledLink with the id of "ImageCard__StyledLink-sc-1s2sl3h-0".
Consider using the attrs method, together with a style object for frequently changed styles.
Example:
  const Component = styled.div.attrs(props => ({
    style: {
      background: props.background,
    },
  }))`width: 100%;`

  <Component />
  ```
  
  If so, could we do like the example and move it up there to keep the styles together or does that still break?